src/ElementQueries.js

```diff
@@ -194,9 +194,12 @@
 
         /**
          * Start the magic. Go through all collected rules (readRules()) and attach the resize-listener.
+         *
+         * @param {HTMLElement} container only elements of the container are considered (document.body if not set)
+         *
          */
-        function findElementQueriesElements() {
-            var query = getQuery();
+        function findElementQueriesElements(container) {
+            var query = (container) ? container.querySelectorAll.bind(container) : getQuery();
```

Comment:
No, jquery or mootools are used when document.querySelectorAll is not defined. See https://github.com/marcj/css-element-queries/blob/134443d31c10ef8c0b475b892c2ecc4f75334dc5/src/ElementQueries.js#L185-L187
So either we drop support for that or we continue to support both.